lib/std/builtin.zig

```diff
@@ -91,6 +91,17 @@ pub const AtomicRmwOp = enum {
     Min,
 };
 
+/// This data structure is used by the Zig language code generation and
+/// therefore must be kept in sync with the compiler implementation.
+pub const CodeModel = enum {
+    Default,
+    Tiny,
+    Small,
+    Kernel,
+    Medium,
+    Large,
+};
```

Comment:
New enums should follow #2101. This will be nice because you can use `@tagName` above instead of the switch.